src/sentry/api/endpoints/organization_integration_repository_project_path_configs.py

```diff
@@ -1,13 +1,70 @@
 from __future__ import absolute_import
 
+from rest_framework import status, serializers
 
 from sentry.api.bases.organization import OrganizationIntegrationsPermission
 from sentry.api.bases.organization_integrations import OrganizationIntegrationBaseEndpoint
 from sentry.api.serializers import serialize
-from sentry.models import RepositoryProjectPathConfig
+from sentry.api.serializers.rest_framework.base import CamelSnakeModelSerializer
+from sentry.models import RepositoryProjectPathConfig, Project, Repository
 from sentry.utils.compat import map
 
 
+class RepositoryProjectPathConfigSerializer(CamelSnakeModelSerializer):
+    repository_id = serializers.IntegerField(required=True)
+    project_id = serializers.IntegerField(required=True)
+    stack_root = serializers.CharField(required=True)
```

Comment:
will stack_root and source_root work if we pass in `""`? I'm not sure if we can make these required currently

Reply:
@MeredithAnya I should have a test for that